plugin.json

```diff
@@ -21,6 +21,13 @@
     "settings_schema": {
         "header": "The GitHub plugin for Mattermost allows users to subscribe to notifications, stay up-to-date with reviews, see the status of pull requests at a glance, and other common GitHub actions - directly from Mattermost. \n \n Instructions for setup are [available here](https://www.mattermost.com/pl/default-github-plugin#configuration).",
         "settings": [
+            {
+                "key": "UsePreregisteredApplication",
+                "display_name": "Use Preregistered OAuth Application:",
+                "type": "bool",
+                "help_text": "When true, instructs plugin to use preregistered GitHub OAuth application - application registration steps can be skipped. Requires Chimera Proxy URL to be configured for the server.",
```

Comment:
I wonder if we should add more context to this. Maybe a link to what is Chimera and what are the implications of using Chimera. @aaronrothschild thoughts?

Reply:
Yes, for sure. Chimera is not yet available for users so we do not have any documentation yet, but I think adding some link to documentation could be a way to go if we are comfortable hardcoding URL here.

Reply:
I don't expect documentation to change much their links, so I would go that direction. Also, this configuration will be visible for all users (including on-prem), so I feel it would be nice for them to know what is chimera.

I would like it even better if this setting is only visible for cloud, but that would mean custom settings, and I am not a big fan of those.